projectsmember/Pages/CooperationRequest/Profiles/HamidForoughnia.html

```diff
@@ -0,0 +1,165 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title> رزومه</title>
```

Comment:
title is one point we give as much as info as we can to the search engines, 
maybe something like: soja.irish | Hamid Foroughnia's Online Resume 
be a better choice!